diff --git a/node_modules/react-native-sound/RNSound.podspec b/node_modules/react-native-sound/RNSound.podspec
index 1432387..bf96269 100644
--- a/node_modules/react-native-sound/RNSound.podspec
+++ b/node_modules/react-native-sound/RNSound.podspec
@@ -12,10 +12,11 @@ Pod::Spec.new do |s|
 
   s.authors      = package['author']
   s.homepage     = package['homepage']
-  s.platform     = :ios, "9.0"
+  s.platform     = :ios, "12.0"
 
   s.source       = { :git => "https://github.com/zmxv/react-native-sound.git" }
   s.source_files = "ios/**/*.{h,m,mm,cpp}"
+  s.frameworks   = "AVFoundation"
 
   # Use install_modules_dependencies helper to install the dependencies if React Native version >=0.71.0.
   # See https://github.com/facebook/react-native/blob/febf6b7f33fdb4904669f99d795eba4c0f95d7bf/scripts/cocoapods/new_architecture.rb#L79.
